webpack/assets/javascripts/react_app/components/charts/StatisticsChartsList.js

```diff
@@ -26,7 +27,7 @@ class StatisticsChartsList extends React.Component {
   render() {
     const noDataMsg = __('No data available').toString();
     const tip = __('Expand the chart').toString();
-    const charts = this.props.charts.map(chart => {
+    const charts = map(this.props.charts, chart => {
```

Comment:
when the state was an array, array.map is used.
after this commit the state will be an object key/value, so lodash map is used